test-main.js

```diff
@@ -5,6 +5,7 @@ var TEST_REGEXP = /tests\/.*\.js$/i;
 var realSinon = sinon;
 beforeEach(function() {
     sinon = realSinon.sandbox.create();
+    localStorage.clear();
```

Comment:
This is a pretty big change. Are we sure it's not going to adversely affect other tests that use localstorage?